dsp_permissions_scripts/models/host.py

```diff
@@ -3,10 +3,7 @@ class Hosts:
 
     LOCALHOST = "http://0.0.0.0:3333"
     PROD = "https://api.dasch.swiss"
-    RDU_STAGE = "https://api.rdu-stage.dasch.swiss"
     DEV = "https://api.dev.dasch.swiss"
-    LS_PROD = "https://api.ls-prod.admin.ch"
-    STAGE = "https://api.stage.dasch.swiss"
```

Comment:
These 3 are not really necessary. I always use `Host.get_host("ls-prod")` etc. Now, I have only left in place the ones that are needed in `authentication.py`